### ISSUE TITLE
Blazor GetJsonAsync returns an empty tuple

### ISSUE THREAD

**Issue by MaurizioVerde** [aspnet/AspNetCore#12723]
As for https://github.com/aspnet/AspNetCore/issues/11144

In version SDK 3.0.100-preview7-012821 the GetJsonAsync extension method returns an empty tuple
To Reproduce
    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.0.0-preview7.19365.7" />
    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="3.0.0-preview7.19365.7" PrivateAssets="all" />

Run this code 

var retValue = await Http.GetJsonAsync<(bool Success, string ErrorMessage, MyClass MyClass)>("api/MyClass/GetMyClass");

the APIController returns (true, null, MyClass)
The returned tuple from GetJsonAsync is empty (false, null, null)

In preview 5 before version '3.0.0-preview6.19307.2' the tuple was successfully deserialized

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us, @MaurizioVerde. This behavior change is caused by the change of the serializer we use. If you are looking for a feature from this serializer, you can file a new issue for the team who owns it at: https://github.com/dotnet/corefx

**Comment by mkArtakMSFT** [comment]
To unblock yourself, you can deserialize the data returned form the service to some other type (array ?) and then manually handle the conversion.

**Comment by MaurizioVerde** [comment]
> To unblock yourself, you can deserialize the data returned form the service to some other type (array ?) and then manually handle the conversion.

Thanks @mkArtakMSFT,  worked around this way

@using Newtonsoft.Json
@using Newtonsoft.Json.Serialization
@using System.Text.Json.Serialization
        var message = await Http.GetAsync("api/Seminar/GetFirstAvailable");
        string content = await message.Content.ReadAsStringAsync();
        (bool Success, string ErrorMessage, Seminar Seminar) retValue = JsonConvert.DeserializeObject<(bool Success, string ErrorMessage, Seminar Seminar)>(content);

(I had to go back to Newtonsoft.Json due to  "options.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore" missing in System.Text.Json)
